discounts/rust/shipping-discounts/default/shopify.function.extension.toml.liquid

```diff
@@ -1,5 +1,6 @@
 name = "{{name}}"
 type = "{{extensionType}}"
+buildWasmPath = "target/wasm32-wasi/release/{{name | replace: " ", "-" | downcase}}.wasm"
```

Comment:
CLI 3.0 can read both camelCase and snake_case in the toml file, but I'd recommend sticking with snake_case for consistency

Reply:
For `buildWasmPath`? That's a built-in arg to the CLI now